### ISSUE TITLE
Lazy filtering cause empty combo-box after readBean() 

### ISSUE THREAD

**Issue by releasedin** [vaadin/vaadin-combo-box-flow#233]
Steps to reproduce
1. Filter the combo-box with a large data provider (to have lazy loading)
2. Select one of the filtered items (**select only with mouse**, not using arrows)
3. Execute readBean function

Result: **The value from combo-box disappear.**

![lazy-combo-box-bug](https://user-images.githubusercontent.com/20513656/54475596-8c6fa080-47fb-11e9-8f39-517c610e672e.gif)

This is a piece of code to reproduce: 

A simple bean:
```
  public class TestBean {

        private String value;

        public String getValue() {
            return value;
        }

        public void setValue(String value) {
            this.value = value;
        }
    }
```
Component with a binder:
```
        Binder<TestBean> binder = new BeanValidationBinder<>(TestBean.class);
        binder.setBean(new TestBean());

        List<String> list = new ArrayList<>();
        for (int i = 0; i <= 2000; i++) {
            list.add(String.valueOf(i));
        }

        ComboBox<String> comboBox = new ComboBox<>();
        comboBox.setItems(list);
        binder.forField(comboBox).bind(TestBean::getValue, TestBean::setValue);

        Button button = new Button("Read Bean");
        button.addClickListener(e -> {
            TestBean bean = binder.getBean();
            //...
            binder.readBean(bean);
            // Here, on the first click the binder has value,
            // on the next click the value is null
        });

        HorizontalLayout container = new HorizontalLayout(comboBox, button);
```

Vaadin version: 12.0.7 and 13.0.1
Browsers: Chrome 72, Firefox 65.0.2

**Comment by pekam** [comment]
Hi, and thanks for the clear description!

This bug was actually fixed a little while ago.

Closing as a duplicate of the fixed issue https://github.com/vaadin/vaadin-combo-box-flow/issues/216

**Comment by releasedin** [comment]
Thanks for revision
Can I ask when the fix will be released?
I have this bug in production and the client can't wait.

**Comment by pekam** [comment]
There should be a maintenance release coming later this week.

**Comment by pekam** [comment]
Unfortunately it was not released yet, hopefully early next week.

**Comment by pekam** [comment]
Released in `vaadin-combo-box-flow 2.1.3`
